one.html

```diff
@@ -0,0 +1,39 @@
+<head>
+    <title>Vue</title>
+</head>
+<body>
+   <div id='app'>
+      <h1 style="color: brown; padding:3%;">{{ header.toUpperCase() }}</h1>
+      <img src="one.jpeg" style="padding-left: 0ch;"/><br><br>
+    
+      <input type="text" v-model="newitem" placeholder="add an items">
+      <button style="color:rgb(177, 71, 122)" :disabled="newitem.length===0" class="btn btn-primary" @click="save">save item</button><br>
+      <a href="https://www.justdial.com/Davangere/Vishal-Mega-Mart/9999P8192-8192-181128115038-B2Y5_BZDET">click here for more info</a>
+      <ul style="color:green; text-decoration: darkgreen;padding: 2%;">
```

Comment:
@janardhanakulakarni same here.

Reply:
I have created the separate css file please look at it.